packages/js/onboarding/src/components/WooPaymentsMethodsLogos/PaymentMethodsLogos.scss

```
@@ -2,12 +2,20 @@
 	display: flex;
 	align-items: center;
 	padding: $gap-small 0;
+	gap: 8px;
```

This was added to replace `margin-right: 8px;` defined next. One of the reasons why is that `margin-right` adds it to the last element too, but isn't needed.

---

That's a very good change! I suggest using the defined gap props.
```suggestion
	gap: $gap-smaller;
```